ios/AirshipReactNative.swift

```diff
@@ -328,10 +328,10 @@ public extension AirshipReactNative {
 // Actions
 @objc
 public extension AirshipReactNative {
-    func actionsRun(actionName: String, actionValue: Any?) async throws-> Any? {
+    func actionsRun(action: [String: Any?]) async throws-> Any? {
         return try await AirshipProxy.shared.action.runAction(
-            actionName,
-            value: try AirshipJSON.wrap(actionValue)
+            action["name"] as! String,
```

Comment:
you will want to throw here instead of force unwrapping.

```
guard let name = action["name"] as? String else { throw AirshipErrors.error("missing name") }